docs/environment_creation_from_scratch.md

```diff
@@ -1,54 +1,112 @@
 # Creating a new environment from scratch
 
 When setting up a new environment:
-1. DNS records will need to be created by DLUHC for certificate validation and to point at the cloudfront distribution and app load balancer
-1. Secret values will need to be filled in manually
-1. The meta account will need updating with additional roles access to the ecr, since this is currently manual
-1. A new deployment pipeline will need to be setup
+1. DNS records will need to be added by DLUHC for certificate validation and to point at the cloudfront distribution and 
+app load balancer
+2. Secret values will need to be filled in manually
+3. The meta account will need updating with additional roles access to the ecr, since this is currently manual
+4. A new deployment pipeline will need to be setup
+
+See the process below for detailed steps.
 
 ## The process
 
+### Create a new terraform folder
+
+If the environment hasn't been created / defined before:
+
+In the terraform folder, add a new sub-folder with the name of the environment.
+
+Add a `main.tf` file to the folder, and define any infrastructure here (e.g. modules to use). You may also add 
```

Comment:
You could maybe also call out any local vars that need to be set correctly e.g. `provider_role_arn` (and maybe `app_host` and `load_balancer_domain_name`). Also probs fine without.

Reply:
Added 👍